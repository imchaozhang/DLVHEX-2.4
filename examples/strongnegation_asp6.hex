% specific head literal parser test

q.

a v b :- q.
c v -d :- q.
-b v -c :- q.
-b v -c v -d v e :- q.
a v -b v -c v -d v e :- q.
