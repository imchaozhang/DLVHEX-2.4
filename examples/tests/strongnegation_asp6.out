{q, b, -d, -c}
{q, a, -d, -c}
{q, a, c, -b}
{q, a, -d, -b}
